src/main/java/enums/Attribute.java

```diff
@@ -1,5 +1,6 @@
 package enums;
 
 public enum Attribute {
-    HREF
+    HREF,
+    SRC
```

Comment:
Minor - In java when there is no constructor for enum, enums values should be defined in one line.
HREF, SRC